packages/http-client-java/generator/http-client-generator-test/src/main/java/com/type/property/additionalproperties/models/ExtendsUnknownAdditionalProperties.java

```diff
@@ -29,7 +30,7 @@ public class ExtendsUnknownAdditionalProperties implements JsonSerializable<Exte
      * The model extends from Record<unknown> type.
      */
     @Generated
-    private Map<String, Object> additionalProperties;
+    private Map<String, BinaryData> additionalProperties;
```

Comment:
I'm quite confused. `use-object-for-unknown: true` means we generate `BinaryData` for unknown types? What does `object` here means?

Reply:
`true` means use `Object`.

Default is `false` and `BinaryData`.

This change is due to this `false`.